core/src/main/scala/frameless/InjectionEnum.scala

```diff
@@ -0,0 +1,54 @@
+package frameless
+
+import scala.reflect.{ClassTag, classTag}
+import shapeless._
+
+object InjectionEnum {
+  type InjectionEnum[A] = Injection[A, String]
```

Comment:
I would inline this type alias, then tend to leak in public API and make them more convoluted...